libs/eavmlib/src/esp.erl

```diff
@@ -132,6 +133,21 @@ sleep_enable_ext0_wakeup(_Pin, _Level) ->
 sleep_enable_ext1_wakeup(_Mask, _Mode) ->
     erlang:nif_error(undefined).
 
+%%-----------------------------------------------------------------------------
+%% @param   Namespace NVS namespace
+%% @param   Key NVS key
+%% @returns tagged tuple with binary value associated with this key in NV
+%%          storage, {error, not_found} if there is no value associated with
+%%          this key, or in general {error, Reason} for any other error.
+%% @doc     Get the binary value associated with a key, or undefined, if
+%%          there is no value associated with this key.
+%% @end
+%%-----------------------------------------------------------------------------
+-spec nvs_fetch_binary(Namespace :: atom(), Key :: atom()) ->
+    {ok, binary()} | {error, not_found} | {error, atom()}.
+nvs_get_binary(Namespace, Key) when is_atom(Namespace) andalso is_atom(Key) ->
```

Comment:
The spec doesn’t match the function name (?)

Reply:
Copy & paste mistake. Fixed.